### ISSUE TITLE
[Content group banner/Content item horizontal]: QA: The information conveyed visually through the icons is not available for the screen reader users

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9653]
### Description

Page Element:
CTA types like External, download, video ...

Issue Description:
The information conveyed visually through these icons is not available for the screen reader users
Note: Same issue is observed in Content item horizontal

Recommend Fix:
The information conveyed visually should be available for the screen reader user as well. 
a generic example: 
1. feed back form opens in new window
2. products opens external link
3. Annual report downloads PDF

User Impact:
When information is conveyed only through icons, images or graphics and has no appropriate text alternative, screen reader users are unable to understand it.

Tool:
Voice Over_iPhone
JAWS_Desktop

### Component(s) impacted

Content group banner
Content item horizontal

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/?path=/story/components-content-block-media--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
